src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -445,6 +438,84 @@ extendedKeysOptions()
   return fmt_output.str();
 }
 
+void
+SubCmdValidate::print_help_internal() const
+{
+  if (m_device.empty())
+    printHelp(false, extendedKeysOptions());
+  else {
+    const std::string deviceClass = XBU::get_device_class(m_device, true);
+    auto it = jsonOptions.find(deviceClass);
+  
+    XBUtilities::VectorPairStrings help_tests = { all_test };
+    if (it != jsonOptions.end() && it->second.size() > 3)
+      help_tests.emplace_back(quick_test);
+
+    static const std::string testOptionValues = XBU::create_suboption_list_map(deviceClass, jsonOptions, help_tests);
+    std::vector<std::string> tempVec;
+    common_options.add_options()
+      ("report,r", boost::program_options::value<decltype(tempVec)>(&tempVec)->multitoken(), (std::string("The type of report to be produced. Reports currently available are:\n") + testOptionValues).c_str() )
+    ;
+    printHelp(common_options, m_hiddenOptions, deviceClass, false, extendedKeysOptions());
+  }
+}
+
+static void
+set_test_data(std::vector<std::shared_ptr<TestRunner>>& collection,
+              std::shared_ptr<TestRunner> test,
+              const std::vector<std::string>& param,
+              const std::string& xclbin_path)
+{
+  if (!param.empty() && boost::equals(param[0], test->get_name())) {
```

Comment:
Be consistent with the curly braces.  Either use them for 1 line of code or not.